tap_snowflake/connection.py

```diff
@@ -80,7 +80,7 @@ def open_connection(self):
                 client_prefetch_threads=self.connection_config.get(
                     'client_prefetch_threads', 4),
                 insecure_mode=self.connection_config.get('insecure_mode', False),
-                network_timeout=50
+                network_timeout=1800
```

Comment:
network_timeout was applied to query execution, failing after 50seconds.
<img width="1509" alt="image" src="https://github.com/symon-ai/pipelinewise-tap-snowflake/assets/86244437/9518f73d-5aba-4d52-9a75-7930c01edf9f">

Select * query for 30cols 60M rows table Raga was testing took around 1min 11s, increasing the timeout to 30mins.